indexer/packages/postgres/src/loops/vault-refresher.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  stats,
+  NodeEnv,
+} from '@dydxprotocol-indexer/base';
+import _ from 'lodash';
+
+import config from '../config';
+import * as VaultTable from '../stores/vault-table';
+import { Options, VaultFromDatabase } from '../types';
+import { startUpdateLoop } from './loopHelper';
+
+let vaultAddresses: Set<string> = new Set();
```

Comment:
Why are we declaring a singleton cache in Postgres? Shouldn't this live in the actual service (`ender` in this case?)